packages/mobile/src/components/drawers/ConfirmationDrawer.tsx

```diff
@@ -90,34 +80,35 @@ const ConfirmationDrawerContent = (props: DrawerContentProps) => {
   }, [onClose, onCancel])
 
   return (
-    <>
-      <View style={styles.header}>
-        <IconInfo
-          style={styles.headerIcon}
-          height={spacing(5)}
-          width={spacing(5)}
-          fill={neutral}
-        />
-        <Text fontSize='xl' weight='heavy' textTransform='uppercase'>
+    <Flex gap='xl' pv='xl'>
+      <Flex
+        direction='row'
```

Comment:
part of me wishes we went with `<Flex row>` and `<Flex column>`

Reply:
definitely! will add those props next week and migrate us! plus i want to make flex act the same way in both platforms :)